### PR TITLE
Fix for multiple definition of comDat

### PR DESCRIPTION
**Pull request by akvavit01**
tested and worked on Arch Linux

### CODE REVIEW

File: wiringPiD/drcNetCmd.h [hunk 1]
```diff
@@ -34,11 +34,19 @@
 #define	DRCN_DIGITAL_READ8	8
 #define	DRCN_ANALOG_READ	9
 
-
+// Multiple definition for comDat fix
```
Comment (DarrienG) [inline wiringPiD/drcNetCmd.h:37]: Why not just delete this rather than comment it out?

**Review by liffiton — CHANGES_REQUESTED** [review]
Apparently I don't know how to use the review mechanism here, because I've had this review "pending" for months now suggesting exactly what DarrienG did.  But I didn't realize that "pending" meant it wasn't visible to anyone but me yet...

File: wiringPiD/drcNetCmd.h (hunk 1 above)
Comment (liffiton) [reply wiringPiD/drcNetCmd.h]: Yeah, I agree.  I'm very confused by Github's pull request / review interface at this point, but I'm going to try to get this thing merged with the extra definition deleted.  Apologies if things get more mangled along the way.  :)

**Review by liffiton — APPROVED** [review]

**Comment by liffiton** [comment]
Sorry that took so long...  I think I have a bit better grasp on Github's UI for reviews now.

**Pull request merged**